types/d3/v3/d3-tests.ts

```diff
@@ -786,7 +786,7 @@ function populationPyramid() {
         .scale(y)
         .orient("right")
         .tickSize(-width)
-        .tickFormat(function (d) { return Math.round(d / 1e6) + "M"; } );
+        .tickFormat(function (d, i) { console.log(i); return Math.round(d / 1e6) + "M"; } );
```

Comment:
Since the tests aren't actually run as code, you just need to verify the correct type is received. You can do that with an `// $ExpectType` comment:

```suggestion
        .tickFormat(function (d, i) {
            // $ExpectType number
            i;
            return Math.round(d / 1e6) + "M";
        });
```

Reply:
Thanks for the suggestion. I have updated it.